evaluation_script/main.py

```diff
@@ -186,28 +196,36 @@ def evaluate(test_annotation_file, user_submission_file, phase_codename, **kwarg
     recall = 0
     fpr = 0
     f1 = 0
+    qa_precision = 0
+    qa_recall = 0
+    qa_fpr = 0
+    qa_f1 = 0
+
     if phase_codename == "unsupervised" or phase_codename == "supervised" or phase_codename == "final":
         print("evaluating for " +phase_codename+ " phase")
         extension = get_file_extension(user_submission_file)
         if extension == "tsv" or extension == "gz": 
             (tp, tn, fp, fn) = calculate_base_metrics(user_submission_file, truth)
-            precision = calculate_precision(tp, fp)
-            recall = calculate_recall(tp, fn)
-            fpr = calculate_recall(fp, tn)
-            f1 = calculate_f1(precision, recall)
-        else:
-            precision = 0
-            recall = 0
-            fpr = 0
-            f1 = 0
+            if not are_all_base_metrics_zero(tp, tn, fp, fn):
```

Comment:
Same question as line 26 - Why isn't the `are_all_base_metrics_zero` test occurring inside the metric calculations?

Reply:
Same response as line 26. Let me know if that makes sense.